v2/yarpchttp/handler.go

```diff
@@ -238,8 +239,10 @@ func (rw *responseWriter) Close(httpStatusCode int) {
 	// since writing to the response body is an implicit 200.
 	rw.w.WriteHeader(httpStatusCode)
 	if rw.buf != nil {
-		// TODO: log this error?
-		_, _ = rw.buf.WriteTo(rw.w)
+		_, err := rw.buf.WriteTo(rw.w)
+		if err != nil {
+			rw.logger.Warn("error writing gRPC response during close", zap.Error(err))
```

Comment:
I'm assuming we're using `Warn` in light of the ongoing default severity discussion?

Reply:
Honestly, it seemed like the right level regardless. We should audit levels later.